### PR TITLE
Property default value generation updated in PythonHL code generator

### PR DESCRIPTION
**Pull request by mesnehal**
Updated the PythonHL code generator for properties default value generation with respect to the issue https://github.com/tango-controls/pogo/issues/78

### CODE REVIEW

**Comment by mesnehal** [comment]
Updated the default value generation in PythonHL code for device and class properties of various data types. Attached the before and after code generation example. 
[AntonTesterAfter.zip](https://github.com/tango-controls/pogo/files/4084761/AntonTesterAfter.zip)
[AntonTesterBefore.zip](https://github.com/tango-controls/pogo/files/4084762/AntonTesterBefore.zip)

File: fr.esrf.tango.generator.xtend/src/fr/esrf/tango/pogo/generator/python/PythonUtils.xtend [hunk 1]
```diff
@@ -83,6 +83,10 @@ class PythonUtils {
 	    	{
 	    		return "[\"" + str.substring(1, str.length()-1).replaceAll(", ",",").replaceAll(",","\", \"") + "\"]";
 	    	}
+	    	else if (str.contains(" "))
+	    	{
+	    		return "[\"" + str.substring(1, str.length()-1).replaceAll(" "," ")+"\"]";
```
Comment (ajoubertza) [inline fr.esrf.tango.generator.xtend/src/fr/esrf/tango/pogo/generator/python/PythonUtils.xtend]: Does `replaceAll(" "," ")` do anything useful?  Replacing spaces with spaces seems unnecessary.

Comment (mesnehal) [reply fr.esrf.tango.generator.xtend/src/fr/esrf/tango/pogo/generator/python/PythonUtils.xtend]: Agree. No need to replace space. done the change.

**Comment by mesnehal** [comment]
@ajoubertza The generation works for properties with empty default values. The suggested change is merged.

**Comment by NexeyaSGara** [comment]
Ok for me

**Comment by Pascal-Verdier** [comment]
Will be available in next Pogo release

**Pull request merged**